JavaShared/src/main/java/org/openzen/zenscript/javashared/prepare/JavaPrepareDefinitionVisitor.java

```diff
@@ -33,6 +34,24 @@ public class JavaPrepareDefinitionVisitor implements DefinitionVisitor<JavaClass
 	private final JavaClass outerClass;
 
 	{
+		{
+			JavaNativeClass cls = new JavaNativeClass(new JavaClass("java.util", "Set", JavaClass.Kind.INTERFACE));
+			cls.addInstanceMethod("add", "add", "(Ljava/lang/Object;)Z");
+			cls.addInstanceMethod("remove", "remove", "(Ljava/lang/Object;)Z");
+			cls.addInstanceMethod("contains", "contains", "(Ljava/lang/Object;)Z");
+			cls.addMethod("toArray", JavaSpecialMethod.LIST_TO_ARRAY);
```

Comment:
Technically, calling a list method for a set is wrong, but since in actuality `toArray(T[])` is defined on Collection, not on list, this is fine?
We could rename the field, though?

Reply:
Renamed to COLLECTION_TO_ARRAY